pkg/resources/provisioning.cattle.io/v1/cluster/mutator.go

```diff
@@ -102,7 +103,7 @@ func (m *ProvisioningClusterMutator) Admit(request *admission.Request) (*admissi
 	listTrace := trace.New("provisioningCluster Admit", trace.Field{Key: "user", Value: request.UserInfo.Username})
 	defer listTrace.LogIfLong(admission.SlowTraceDuration)
 
-	cluster, err := objectsv1.ClusterFromRequest(&request.AdmissionRequest)
+	cluster, oldCluster, err := objectsv1.ClusterOldAndNewFromRequest(&request.AdmissionRequest)
```

Comment:
Based on the docs of this function and the [return values](https://github.com/rancher/webhook/blob/97679b88dd800c1cbad320721d721e4250fa7a91/pkg/generated/objects/provisioning.cattle.io/v1/objects.go#L38), I think the order is old first, then new. So this should be switched around:

```suggestion
	oldCluster, cluster, err := objectsv1.ClusterOldAndNewFromRequest(&request.AdmissionRequest)
```